docpad.coffee

```diff
@@ -12,7 +12,7 @@ module.exports =
     conf:
       name: 'Vale dos Desenvolvedores'
       description: 'Valley of Developers Meetup Roadmap 2017'
-      date: '12/Set, 25/Out e 11/Nov de 2017'
+      date: '25/Out e 11/Nov de 2017'
```

Comment:
We can keep the passed dates here, I think, since it's a _roadmap_.
You can ~~strikethrough~~ it to

Reply:
It is not relevant since we have no "past events" section.

Reply:
yet ;)